package/patterns-yast.spec

```diff
@@ -51,107 +50,124 @@ Provides:       pattern-visible()
 
 Requires:       libyui-ncurses-pkg
 Requires:       yast2
+Requires:       yast2-firewall
+Requires:       yast2-packager
+Requires:       yast2-theme
+
+Recommends:     yast2-metapackage-handler
+Recommends:     yast2-support
```

Comment:
when you touch it, I think this one is SLE only ( it allows user to easily generate something called support logs that collect various stuff useful to recognize if system is still fully supported by SUSE ).

Reply:
Yup, good point